textdb/textdb-storage/src/test/java/edu/uci/ics/textdb/storage/LuceneDataWriterReaderTest.java

```diff
@@ -34,7 +34,7 @@ public void setUp() throws ParseException{
         analyzer = new  StandardAnalyzer();
         dataWriter = new LuceneDataWriter(dataStore, analyzer );
         QueryParser queryParser = new QueryParser(
-                TestConstants.ATTRIBUTES_PEOPLE.get(0).getFieldName(), analyzer);
+                TestConstants.ATTRIBUTES_PEOPLE[0].getFieldName(), analyzer);

```

Comment:
The comment is for line 57: Why didn't we use the "containsAllResults()" helper function for this test case?